src/plots/Mesh/MeshAttributes.java

```diff
@@ -35,9 +35,11 @@
 
     public final static int MESHCOLOR_FOREGROUND = 0;
     public final static int MESHCOLOR_MESHCUSTOM = 1;
+    public final static int MESHCOLOR_MESHRANDOM = 2;
```

Comment:
It looks like maybe I neglected to update this file when I was adding the *random* mesh coloring option?